fontbe/src/hvar.rs

```diff
@@ -52,30 +51,27 @@ struct AdvanceWidthDeltas {
     /// Variation axes
     axes: Vec<Axis>,
     /// Sparse variation models, keyed by the set of locations they define
-    models: IndexMap<BTreeSet<NormalizedLocation>, VariationModel>,
+    models: HashMap<BTreeSet<NormalizedLocation>, VariationModel>,
```

Comment:
The only reason I was using IndexMap in this piece of code was to be able to retrieve the first() key/value pair in the map, in the global_locations() method which I have deleted as no longer used. The order doesn't matter in this case because it's simply a cache of pre-built variation models